src/config/context.rs

```diff
@@ -123,6 +124,26 @@ fn build_object_store(cfg: &schema::SeafowlConfig) -> Arc<dyn ObjectStore> {
                 ));
             }
 
+            Arc::new(store)
+        }
+        #[cfg(feature = "object-store-gcs")]
+        schema::ObjectStore::GCS(GCS {
+            bucket,
+            google_application_credentials,
+        }) => {
+            let gcs_builder: GoogleCloudStorageBuilder =
+                GoogleCloudStorageBuilder::new().with_bucket_name(bucket);
+
+            let gcs_builder = if let Some(path) = google_application_credentials {
+                gcs_builder.with_service_account_path(path)
+            } else {
+                gcs_builder
```

Comment:
It seems that this option [only works](https://github.com/apache/arrow-rs/blob/master/object_store/src/gcp/mod.rs#L1167-L1173) if Seafowl is running on GCP itself, such that the underlying client can [fetch the credentials](https://github.com/apache/arrow-rs/blob/master/object_store/src/gcp/credential.rs#L326-L333) from the metadata server.

This is fine in that case, though maybe we could just change the wording in the warning message then.

Reply:
> we could just change the wording in the warning message then.
Thanks for the clarification! I [tweaked](https://github.com/splitgraph/seafowl/pull/379/files#diff-b1d0a32c21852f4c587c36f2e476a8966f37ce0973c58f04a28552f3645afbdaR355) the wording.